docs/generatedcode/python.md

```diff
@@ -6,3 +6,147 @@ type: markdown
+        )
+    self.SuggestFortune = channel.unary_unary(
+        '/example.FortuneTeller/SuggestFortune',
+        request_serializer=fortune_pb2.SuggestionRequest.SerializeToString,
+        response_deserializer=fortune_pb2.SuggestionResponse.FromString,
+        )
+
+
+class FortuneTellerServicer(object):
+
+  def TellFortune(self, request, context):
+    """Returns the horoscope and zodiac sign for the given month and day.
+    errors: invalid month or day, fortune unavailable
+    """
+    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+    context.set_details('Method not implemented!')
+    raise NotImplementedError('Method not implemented!')
+
+  def SuggestFortune(self, request, context):
+    """Replaces the fortune for the given zodiac sign with the provided
+one.
+    errors: invalid zodiac sign
+    """
+    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+    context.set_details('Method not implemented!')
+    raise NotImplementedError('Method not implemented!')
+
+
+def add_FortuneTellerServicer_to_server(servicer, server):
+  rpc_method_handlers = {
+      'TellFortune': grpc.unary_unary_rpc_method_handler(
+          servicer.TellFortune,
+          request_deserializer=fortune_pb2.HoroscopeRequest.FromString,
+          response_serializer=fortune_pb2.HoroscopeResponse.SerializeToString,
+      ),
+      'SuggestFortune': grpc.unary_unary_rpc_method_handler(
+          servicer.SuggestFortune,
+          request_deserializer=fortune_pb2.SuggestionRequest.FromString,
+          response_serializer=fortune_pb2.SuggestionResponse.SerializeToString,
+      ),
+  }
+  generic_handler = grpc.method_handlers_generic_handler(
+      'example.FortuneTeller', rpc_method_handlers)
+  server.add_generic_rpc_handlers((generic_handler,))
+```
+
+## Code Elements
+
+The gRPC code generator starts by importing `grpc` package and the plain `_pb2`
+module, synthesized by `protoc`, which defines non-gRPC-specifc code elements,
+like the classes corresponding to protocol buffers messages and descriptors
+used by reflection.
+
+For each service `~` in the `.proto` file, three primary elements are
+generated:
+
+- [**Stub**](#stub): `~Stub` used by the client to connect to a gRPC service.
+- [**Servicer**](#servicer): `~Servicer` used by the server to implement a
+  gRPC service.
+- [**Registration Function**](#registration-function):
+  `add_~Servicer_to_server` function used to register a servicer with a
+  `grpc.Server` object.
+
+
+### Stub
+
+<a name="stub"></a>The generated `Stub` class is used by the gRPC clients.  It
+will have a constructor that takes a `grpc.Channel` object and initializes the
+stub.  For each method in the service, the initializer adds a corresponding
+attribute to the stub object with the same name.  Depending on the RPC type
+(unary or streaming), the value of attribute will be the output of
+`unary_unary`, `unary_streaming`, or `stream_stream` returned from the
```

Comment:
There should be four methods named on this line, right? `unary_unary`, `unary_stream`, `stream_unary`, and `stream_stream`?